flask_server.py

```diff
@@ -424,5 +404,16 @@ def list_filename_variants(file_name):
             os.path.splitext(os.path.basename(file_name))[0]]
 
 
+@app.route("/get_features_list", methods=["GET"])
```

Comment:
I'd just call this `features_list`, so the action is "GET `features_list`", not "GET `get_features_list`".